documentation/oracle/governance/adding_a_price_identifier.md

```diff
@@ -0,0 +1,48 @@
+# Adding a Price Identifier
+
+## What is a price identifier?
+
+A [price identifier](../../synthetic_tokens/glossary.md#price-identifier) is a natural language descriptor of the reference index determining how much collateral is needed for a token sponsor to be properly collateralized. 
+Because DVM voters need to be able to vote on the value of this price identifier when disputes are raised, the DVM keeps a list of approved price identifiers. 
+
+An example of a price identifier is “GOLD_APR20”. 
+This price identifier would be documented in the UMIP that was approved by the UMA tokenholders. 
+That UMIP would contain more information about how to determine the price identifier. 
+
+## Who controls the list of approved price identifiers?
+For each deployment of the DVM, the list of approved price identifiers is controlled by the `IdentifierWhitelist` contract.
+In local and testnet deployments of the DVM, `IdentifierWhitelist` is controlled by a single private key. 
+In the mainnet deployment of the DVM, `IdentifierWhitelist` is controlled by a decentralized governance process, as described below.
+
+## Adding a price identifier to a local deployment
+In a local deployment, your private key controls the `IdentifierWhitelist` contract. 
+You can therefore add any price identifier desired using the `IdentifierWhitelist.addSupportedIdentifier`, as described in step 5 of this [tutorial](../../synthetic_tokens/tutorials/creating_from_truffle.md). 
+
+<!-- TODO: Add a section for ## Adding a price identifier to a testnet deployment --> 
+
+## Adding a price identifier to the mainnet deployment
+The `IdentifierWhitelist` contract in the mainnet deployment of the UMA DVM is controlled by a decentralized governance process. 
+To add a new price identifier, UMA token holders must vote and approve the identifier. 
+
+- Step 1: Discuss
```

Comment:
```suggestion
###Step 1: Discuss
```

Reply:
@rcai1 should we make all the steps sub subsections?

Reply:
I don't think that's necessary. As an aesthetic, I also don't like the idea of it polluting the sidebar on the right with many headers. Do you agree?